src/client/index.ts

```diff
@@ -105,36 +105,28 @@ export class Aggregate<K extends Key, ID extends string> {
   }
   /**
    * Returns the rank/offset/index of the given key, within the bounds.
-   * Specifically, it returns the index of the first item with a key >= the given key.
+   * Specifically, it returns the index of the first item with
+   *
+   * - key >= the given key if `order` is "asc" (default)
+   * - key <= the given key if `order` is "desc"
    */
-  async offsetOf(
+  async indexOf(
```

Comment:
this all looks good.

an additional thing i found i wanted:

when using TableAggregate, you may want the index/offset of a document, without having to calculate the sort key & id.

can we make a method on TableAggregate that takes in a document (and bounds and order) instead of a key & id? preferably with a slightly different name like `indexOfDoc` so you could use `indexOf` when you only have a key.